experimental/FTheoryTools/test/tate_models.jl

```diff
@@ -128,39 +128,39 @@ istar0_s_auxiliary_base_ring, (a1pp, a2pp, a3pp, a4pp, a6pp, vp, wp, mp) = QQ["a
 tate_auxiliary_base_ring, (a1p, a2p, a3p, a4p, a6p, v, w) = QQ["a1p", "a2p", "a3p", "a4p", "a6p", "v", "w"];
 
 # construct Tate models over arbitrary base
-t_istar0_s = global_tate_model(istar0_s_auxiliary_base_ring, [1 2 3 4 6 0 2; -1 -2 -2 -3 -4 1 -1], 3, [a1pp * (vp^2 + wp)^1, mp * (vp^2 + wp)^1 + a2pp * (vp^2 + wp)^2, a3pp * (vp^2 + wp)^2, mp^2 * (vp^2 + wp)^2 + a4pp * (vp^2 + wp)^3, a6pp * (vp^2 + wp)^4]);
```

Comment:
Why have the gradings changed? (At least it looks so to me right now.)

Reply:
All of the gradings in these tests were incorrect in multiple ways:

1. The number of columns in the grading matrices did not match the number of coordinates of the base. This also indicates an "error" of the functions that construct arbitrary bases, as they should probably require that this be the case. I didn't fix that issue in this PR, but I did update all the gradings in these tests.
2. Even setting that aside, the gradings were not updated when we changed the tests to use gauge divisor `v^2 + w = 0` rather than just `w = 0`. Where previously a given variable had grade `n` under `[w]`, now it really should have grade `2 n` under `[v]`, and `w` itself should have grade `2` under `[v]`. I've updated the gradings to make this the case.

Reply:
I see. Thx!